src/app/services/actions/permissions-action-creator.spec.ts

```diff
@@ -432,8 +438,8 @@ describe('Permissions action creators', () => {
         {
           type: 'QUERY_GRAPH_STATUS',
           response: {
-            statusText: 'Failed',
-            status: 'An error occurred when unconsenting. Please try again',
+            statusText: 'Unable to dissent',
```

Comment:
Are these strings localised anywhere down the line?

Reply:
No, they were not...
I've added them to the localisation file